samples/bot-sso-adaptivecard/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-bot-sso-adaptivecard-nodejs",
+    "source": "officeDev",
+    "title": "Teams Bot-SSO-Adaptivecard-Nodejs",
+    "shortDescription": "SSO for your Adaptive Cards Universal Actions.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-sso-adaptivecard/nodejs",
+    "longDescription": [
+      "This sample code demonstrates how to get enable SSO authentication for your Adaptive Cards Universal Actions in bot."
```

Comment:
longDescription
This sample code demonstrates how to implement SSO authentication for your Adaptive Cards